src/Common/src/System/Numerics/Hashing/HashHelpers.cs

```diff
@@ -6,12 +6,14 @@ namespace System.Numerics.Hashing
 {
     internal static class HashHelpers
     {
+        public static int RandomSeed { get; } = Guid.NewGuid().GetHashCode();
```

Comment:
If ValueTuple is moved down to CoreLib you might not be able to use the Crypto functions there due to layering.

I think using the Guid to generate a random seed is a good place to start. Having a random hash code gives us the freedom to change it later with a different algorithm if we need to.